### ISSUE TITLE
`html --rootid` no longer works with sec1

### ISSUE THREAD

**Issue by cwickert** [tomschr/daps#686]
### Problem description
When building html in daps daps-3.3.1, `--rootid` no longer works with `<sec1>`. The html file won't get build and index.html becomes a dangling symlink. I could reproduce the problem in different projects, e.g. doc-sle and doc-public-cloud. PDFs on the other hand do get build, as does using chapters as `--rootid`.

### Expected behavior
It should be possible to build html of `<sec1>`, index.html should point to the generated html.

### Steps to reproduce problem
```
cwickert@localhost:~/git/github.com/suse/doc-sle> daps -d DC-SLES-modules html --rootid sec-modules-install
HTML book built with REMARKS=0, DRAFT=no and META=0:
/home/cwickert/git/github.com/suse/doc-sle/build/SLES-modules/html/sec-modules-install/
cwickert@localhost:~/git/github.com/suse/doc-sle> ll build/SLES-modules/html/sec-modules-install/
total 8
drwxr-xr-x 2 cwickert users 4096 12. Okt 15:24 images
lrwxrwxrwx 1 cwickert users   24 12. Okt 15:24 index.html -> sec-modules-install.html
drwxr-xr-x 5 cwickert users 4096  2. Aug 13:43 static
cwickert@localhost:~/git/github.com/suse/doc-sle> file build/SLES-modules/html/sec-modules-install/index.html
build/SLES-modules/html/sec-modules-install/index.html: broken symbolic link to sec-modules-install.html
cwickert@localhost:~/git/github.com/suse/doc-sle> rpm -q daps
daps-3.3.1-150300.14.3.2.noarch
```

**Comment by fsundermeyer** [comment]
@tomschr Is this something you can look into? This looks like a stylesheet prtoblem to me.

RootID is a chapter (everything is OK):

```
> /usr/bin/xsltproc  --xinclude  --stringparam rootid "cha-64bit" \
  --stringparam img.src.path "images/" \
  --stringparam base.dir "/local/git/doc-sle/build/SLES-all/html/cha-64bit/" \
  --stringparam use.id.as.filename "1" \
  /usr/share/xml/docbook/stylesheet/suse2022-ns/xhtml/chunk.xsl \
  /local/git/doc-sle/build/.profiled/x86_64_zseries_power_aarch64_sles/MAIN.SLEDS.xml
Writing /local/git/doc-sle/build/SLES-all/html/cha-64bit/cha-64bit.html for chapter(cha-64bit)
```
RootID is sect1 (xsltproc output goes to STDOUT)

```
> /usr/bin/xsltproc  --xinclude  --stringparam rootid "sec-64bit-runt" \
  --stringparam img.src.path "images/" \
  --stringparam base.dir "/local/git/doc-sle/build/SLES-all/html/cha-64bit/" \
  --stringparam use.id.as.filename "1" \
  /usr/share/xml/docbook/stylesheet/suse2022-ns/xhtml/chunk.xsl \
  /local/git/doc-sle/build/.profiled/x86_64_zseries_power_aarch64_sles/MAIN.SLEDS.xml         
<?xml version="1.0" encoding="UTF-8"?>
<section xmlns="http://www.w3.org/1999/xhtml" class="sect1" id="sec-64bit-runt" data-id-title="Runtime support"><div class="titlepage"><div><div><h2 class="title"><span class="title-number">
```

**Comment by tomschr** [comment]
The design of the stylesheet was always to split at the chapter level. Everything below that isn't supported. You need to build the parent structure, usually the chapter.

I don't know how many writer fall into this trap and how often this is really needed. Would assume most people build the chapter. Maybe daps could/should catch such a case and refuse to build.

Considering that this is quite a lot of work with little benefit, I would see this as WONTFIX.

**Comment by fsundermeyer** [comment]
Fixed with 817c33e. Now attempts to build with a rootid pointing to a sect1 will result in an error.
